### DISCUSSION TITLE
Send chat message from server from Plugin?

### DISCUSSION THREAD

**Discussion by cfpwastaken** [Impostor/Impostor#248] [Q&A] [answered]
Can I do that?
For like eg. Commands in chat.

**Comment by AwesomeKalin** [comment]
I don't see why it's not possible. Skeld.net did it

**Reply by TouseefX** [reply]
https://amodsus.com/resources/categories/plugin-impostor-server.2/

**Reply by miniduikboot** [reply, marked as answer]
To handle a chat command, you can use https://github.com/Impostor/Impostor/blob/master/src/Impostor.Api/Events/Game/Player/IPlayerChatEvent.cs and cancel the event to stop the message from being shared with other players
